scripts/dotnet-cli-build/MsiTargets.cs

```diff
@@ -60,6 +69,20 @@ private static void AcquireWix(BuildTargetContext c)
             ZipFile.ExtractToDirectory(Path.Combine(WixRoot, "WixTools.zip"), WixRoot);
         }
 
+        private static void AcquireCrt(BuildTargetContext c)
+        {
+            if (File.Exists(CrtRedistributablePath))
+            {
+                return;
+            }
+
+            Directory.CreateDirectory(Path.GetDirectoryName(CrtRedistributablePath));
+
+            c.Info($"Downloading VC_redist.{Arch}.{CrtVersion}.exe..");
+
+            DownloadFile($"https://dotnetcli.blob.core.windows.net/build/vc/{Arch}/VC_redist.{Arch}.{CrtVersion}.exe", CrtRedistributablePath);
```

Comment:
@ellismg I am not familiar with our setup technologies but would have thought there would be a way for CLI setup, when invoked, to download the VC++ redist, independent of our download source (the Azure blob). Is that not possible?

For the aircraft case, perhaps we could call out that we will need to download dependencies and would need an internet connection. This is similar to other setups (E.g. VS) where dependencies are downloaded over Internet and VS wouldnt just install if you only were to download the initial web installer.

The primary question is - how are we intending to align our setup?